Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -13,16 +13,16 @@ public string Execute(string email, string password)
 
             if (CommonValidations.ValidateEmail(email))
             {
-                var user = new User(email, password);
-                message = createUserInDatabase(email, password);
+                var user = new User(email, password, Guid.Empty);
```

Comment:
Guid.Empty wtf????

Reply:
It wouldn't be necessary if we create new model UserCredential
UserService can't create ID , but it had to be set in constructor. That was the way we agreed in chat in thursday. But it will be removed

Reply:
This code was deleted